test/article.test.js

```diff
@@ -2,16 +2,51 @@ const chai = require('chai')
 const chaiHttp = require('chai-http')
 const app = require('../server.js')
 
-chai.should()
-chai.use(chaiHttp)
+const should = chai.should()
 
-function checkResponse(res) {
+function postVote(id, vote, old) {
+  return new Promise((resolve) => {
+    chai.request(app)
+            .post('/articles/vote')
+            .send({ id, vote })
+            .end((err, res) => {
+              resolve({ res, old })
+            })
+  })
+}
+
+function getArticleData(id) {
+  return new Promise((resolve, reject) => {
+    chai.request(app)
+            .get(`/articles/${id}`)
+            .end((err, res) => {
+              if (err) reject(err)
```

Comment:
Should be consistent with checking for error, the above function doesn't, you can pick to check or not